src/components/currencyConverter/CurrencyConverter.jsx

```diff
@@ -10,7 +10,14 @@ const CurrencyConverter = ({currencyList, amount, fromCurrency, toCurrency, setN
     const fromOptions = currencyList.map(currency => <Dropdown.Item key={currency} icon={iconMap[currency]}>{currency}</Dropdown.Item>);
     const toOptions = currencyList.filter(currency => currency !== fromCurrency).map(currency => <Dropdown.Item key={currency} icon={iconMap[currency]}>{currency}</Dropdown.Item>);
 
-    const handleAmountChange = (event) => setNewAmount(event.target.value);
+    // const handleAmountChange = (event) => setNewAmount(event.target.value);
+
+    const handleAmountChange = (event) => {
+        if (event.target.value >= 0) {
+            setNewAmount(event.target.value);
+        }
+    }
```

Comment:
rather than commenting out the original `handleAmountChange` function, you could have just amended it